app/views/drawings/_drawings.html.haml

```diff
@@ -1 +1,51 @@
-= render @drawings.desc
+.container
+  %h3 Manage Drawings
+  %table.table.table-bordered.table-condensed.table-hover.table-responsive
+    %thead
+      %tr
+        %th ID
+        %th File
+        %th Subject Matter
+        %th Description
+        %th Age
+        %th Artist's Gender
+        %th Story
+        %th Mood Rate
+        %th Edit
+        %th Delete
+        %th User Admin
+        %th Status
+    %tbody
+    - @drawings.each do |drawing|
+      %tr
+        %td
+          = drawing.id
+        %td
+          = image_tag drawing.image.url(:thumb)
+        %td
+          = drawing.subject_matter
+        %td
+          = drawing.description[0, 29].concat('...')
```

Comment:
@krissy yep sounds good, best to keep the view as pure as possible and if the helper can be reused elsewhere I can't see an issue with that. 

Just thinking is it worth leaving it here for the pilot and creating the helper method after _OR_ quickly doing the helper method now? I know I won't have time to do it before the pilot